doc/source/conf.py

```diff
@@ -0,0 +1,76 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+import datetime
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../../src"))
+
+# -- Project information -----------------------------------------------------
+
+project = "lammmpsio"
+year = datetime.date.today().year
+copyright = f"2021-{year}, Auburn University"
+author = "Michael P. Howard"
+version = "0.7.0"
+release = "0.7.0"
+
+# -- General configuration ---------------------------------------------------
+
+extensions = [
+    "IPython.sphinxext.ipython_console_highlighting",
+    "nbsphinx",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.napoleon",
+    "sphinx_design",
+    "sphinx_favicon",
+    "myst_parser",
+]
+
+
+templates_path = ["_templates"]
```

Comment:
Just noting: you can add the class template like we have in relentless if you don't like the default by adding a template like this (path and name are important):

https://github.com/mphowardlab/relentless/blob/main/doc/source/_templates/autosummary/class.rst